Test/VSComp2010/Problem1-SumMax.dfy.expect

```diff
@@ -1,2 +1,2 @@
 
-Dafny program verifier finished with 3 verified, 0 errors
```

Comment:
I think you have a separate PR for that change right? And I would like to discuss it as a team because I would rather prefer to have imprecise positive feedback (which for example indicates that it's not vacuously verified because it was the wrong file) rather than no positive feedback at all.

Reply:
I agree that the feedback is important. I'm putting this PR on hold until I find a way to get precise positive feedback.